_search-plugins/async/index.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: default
-title: Asynchronous search
+title: Asynchronous search plugin
```

Comment:
Should we remove the `search_plugin` directory and collection? For example, this page would go into the following directory, `_search-query` > `async` > `index.md`. Then, we can place any child pages of "Asynchronous search plugin" into the `async` directory.

Reply:
would that make sense to move all of the search plugins back to Search and query? We'd have to move them all (k-NN, quergy, SQL, async).

just to provide context. . .In the original plan we had these categories under Search and query: 
**Search with SQL**
**Search asynchronously** 
but because they are actual plugins, i thought best to leave them within the **Search plugins** menu. 
but if that makes sense to move all search plugins to Search and query we can do that.

Reply:
They need to be moved under the Search category. We will not have any "x plugin" categories in the nav pane going forward.

Reply:
ok sounds good.